### PR TITLE
Eventbrite: New Actions

### PR DESCRIPTION
**Pull request by michelle0927**
New Actions:

Get Event Summary
Get Event Attendees
Create Event

### CODE REVIEW

File: components/eventbrite/actions/get-event-attendees/get-event-attendees.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    async *attendeeStream(params = {}) {
+      let hasMoreItems;
+      do {
+        const {
+          ["attendees"]: items,
+          pagination = {},
+        } = await this.eventbrite.getEventAttendees(this.eventId, params);
+        for (const attendee of items) {
+          yield attendee;
+        }
```
Comment (jverce) [inline components/eventbrite/actions/get-event-attendees/get-event-attendees.js:23]:
```suggestion
        const {
          attendees,
          pagination = {},
        } = await this.eventbrite.getEventAttendees(this.eventId, params);
        for (const attendee of attendees) {
          yield attendee;
        }
```

File: components/eventbrite/actions/get-event-summary/get-event-summary.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  description: "Get event data for a specified event.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    eventbrite,
+    eventId: { propDefinition: [eventbrite, "eventId"] },
+  },
+  async run() {
+    return await this.eventbrite.getEvent(this.eventId);
+  },
```
Comment (jverce) [inline components/eventbrite/actions/get-event-summary/get-event-summary.js:15]: Shouldn't this return the summary only?
```suggestion
  async run() {
    const { summary } = await this.eventbrite.getEvent(this.eventId);
    return summary;
  },
```

File: components/eventbrite/actions/locales.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "fr_CA",
+  "fr_CH",
+  "fr_FR",
+  "hi_IN",
+  "it_IT",
+  "nl_BE",
+  "nl_NL",
+  "pt_BR",
+  "pt_PT",
+];
```
Comment (jverce) [inline components/eventbrite/actions/locales.js:30]: Is there an endpoint in Eventbrite's API from which we could retrieve these codes? Or are they documented somewhere?

An easier alternative would be to use a package like `country-locale-map` that contains all this info:

```suggestion
const clm = require("country-locale-map");

module.exports = clm
  .getAllCountries()
  .flatMap(({
    alpha2: countryCode,
    name: countryName,
    emoji: countryFlag,
    languages,
  }) => languages.map((language) => ({
    label: `${countryFlag} ${countryName} (${language})`,
    value: `${language}_${countryCode}`,
  })));
```

**Review by jverce — CHANGES_REQUESTED** [review]
Some small changes, but other than that LGTM!

File: components/eventbrite/actions/create-event/create-event.js [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        online_event: this.onlineEvent,
+        organizer_id: this.organizerId,
+        logo_id: this.logoId,
+        venue_id: this.venueId,
+        format_id: this.formatId,
+        category_id: this.categoryId,
+        subcategory_id: this.subcategory_id,
+        listed: this.listed,
+        shareable: this.shareable,
+        invite_only: this.invite_only,
```
[the remark this answers is not in the document]
Comment (michelle0927) [reply components/eventbrite/actions/create-event/create-event.js]: Updated

File: components/eventbrite/actions/get-event-attendees/get-event-attendees.js (hunk 1 above)
Comment (michelle0927) [reply components/eventbrite/actions/get-event-attendees/get-event-attendees.js]: Updated

File: components/eventbrite/actions/get-event-summary/get-event-summary.js (hunk 2 above)
Comment (michelle0927) [reply components/eventbrite/actions/get-event-summary/get-event-summary.js]: Updated

File: components/eventbrite/actions/create-event/create-event.js [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+      type: "string",
+      label: "End Time",
+      description:
+        "The event end time relative to UTC. (Ex. 2018-05-12T02:00:00Z)",
+    },
+    hideStartDate: {
+      type: "boolean",
+      label: "Hide Start Date",
+      description: "Whether the start date should be hidden",
+      optional: true,
```
[the remark this answers is not in the document]
Comment (michelle0927) [reply components/eventbrite/actions/create-event/create-event.js:52]: The documentation includes the defaults of a few of the props, and I determined some of the others through testing. Added this info to the prop descriptions.

File: components/eventbrite/actions/locales.js (hunk 3 above)
Comment (michelle0927) [reply components/eventbrite/actions/locales.js]: There isn't an endpoint for this. Unfortunately, using the country-locale-map includes a lot of languages not supported by Eventbrite and results in errors if they are selected. I added additional labels to the language codes to make them more readable.

**Review by jverce — APPROVED** [review]

**Pull request merged**